tigerpath/majors_and_certificates/majors/COM_2018.json

```
@@ -0,0 +1,263 @@
+                "ANT 4*",
+                "ANT 5*",
+                "ART 3*",
+                "ART 4*",
+                "ART 5*",
+                "CLA 3*",
+                "CLA 4*",
+                "CLA 5*",
+                "ECO 3*",
+                "ECO 4*",
+                "ECO 5*",
+                "HIS 3*",
+                "HIS 4*",
+                "HIS 5*",
+                "PHI 3*",
+                "PHI 4*",
+                "PHI 5*",
+                "POL 3*",
+                "POL 4*",
+                "POL 5*",
+                "PSY 3*",
+                "PSY 4*",
+                "PSY 5*",
+                "REL 3*",
+                "REL 4*",
+                "REL 5*",
+                "SOC 3*",
+                "SOC 4*",
+                "SOC 5*",
+                "WWS 3*",
+                "WWS 4*",
+                "WWS 5*"
+              ]
+            },
+            {
+              "name": "Another Medium",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Comparative work in literature and another medium: two 300-, 400-, or 500-level courses in photography, film, art, art history, architecture or music. In general, these courses involve the same medium (photography, for example); students intending to pair courses involving different media with related content (photography and painting, for example) should obtain written permission from COM’s Director of Undergraduate Studies.",
+              "course_list": [
+                "ART 3*",
+                "ART 4*",
+                "ART 5*",
+                "ARC 3*",
+                "ARC 4*",
+                "ARC 5*",
+                "MUS 3*",
+                "MUS 4*",
+                "MUS 5*",
+                "VIS 3*",
+                "VIS 4*",
+                "VIS 5*"
+              ]
+            },
+            {
+              "name": "Regional or Ethnic Studies",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Comparative work in literature and regional or ethnic studies: two 300-, 400-, or 500-level courses in AFS, AAS, EAP, EAS, ECS, EPS, HLS, JDS, LAS, LAO, NES, or SAS. These two courses normally emerge from a single program, and have the same prefix (AFS, for instance). Alternatively, students on this track might take two 300-, 400-, or 500-level courses in either ENV, GSS, or URB; as elsewhere, courses should emerge from a single program, and have the same prefix. Those interested in these options should consult with COM’s Director of Undergraduate Studies to plan a course of study.",
+              "course_list": [
+                "AFS 3*",
+                "AFS 4*",
+                "AFS 5*",
+                "AAS 3*",
+                "AAS 4*",
+                "AAS 5*",
+                "EAP 3*",
+                "EAP 4*",
+                "EAP 5*",
+                "EAS 3*",
+                "EAS 4*",
+                "EAS 5*",

```

What about ECS?
https://complit.princeton.edu/undergraduate-program/current-majors/requirements
```suggestion
                "EAS 5*",
                "ECS 3*",
                "ECS 4*",
                "ECS 5*",
```